src/EnergyPlus/HVACManager.cc

```diff
@@ -2375,16 +2247,17 @@ void ReportInfiltrations(EnergyPlusData &state)
 
         int NZ = thisInfiltration.ZonePtr;
         auto &thisZoneHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(NZ);
-        auto &thisZone = state.dataHeatBal->Zone(NZ);
+        auto const &thisZone = state.dataHeatBal->Zone(NZ);
```

Comment:
[src/EnergyPlus/HVACManager.cc:2378]:(style),[constVariable],Variable 'thisZone' can be declared as reference to const